bcda/api/requests.go

```diff
@@ -386,17 +387,20 @@ func check429(jobs []models.Job, types []string, version string) ([]string, erro
 				continue
 			}
 
+			// If the job has timed-out we will allow another job to be in progress
+			if time.Now().After(job.CreatedAt.Add(GetJobTimeout())) {
```

Comment:
This is the exact opposite logic that used to exist.

We used to say "If the job HAS NOT timed out, then we'll do stuff". The new logic is "If the job HAS timed out, then we will skip (and allow the job to be worked)"